docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,115 @@
+# How to Understand Strategies Descriptions
+
+Navigating strategies descriptions at [yearn.watch↗](https://yearn.watch/) or [vaults.yearn.finance↗](https://vaults.yearn.finance/) may expose you to terms that you have never seen, this page contains a complete glossary on what each term means when used to describe strategies for yearn vaults.
+
+It complements our DeFi [Glossary↗](https://docs.yearn.finance/resources/defi-glossary) and it's more focused only on the words used by strategy descriptions and shorter explanations.
```

Comment:
This guide focuses primarily on strategy descriptions, but those thirsty for more DeFi jargon can see our broader DeFi [Glossary↗](https://docs.yearn.finance/resources/defi-glossary) here.